doc/source/whatsnew/v0.18.1.txt

```diff
@@ -44,7 +44,8 @@ Enhancements
 API changes
 ~~~~~~~~~~~
 
-
+- ``Period`` and ``PeriodIndex`` now raises ``IncompatibleFrequency`` error
```

Comment:
btw, usually just make these 1 line as the formatter will wrap it (and I think create a weird space otherwise)